bundles/org.openhab.ui.dashboard/ESH-INF/i18n/dashboard_tr.properties

```diff
@@ -5,23 +5,23 @@ index.your-location=Konumunuz\:
 
 setup.subtitle=Kurulum
 setup.welcome=openHAB 2\u2019ye ho\u015fgeldiniz - ba\u015flang\u0131\u00e7 kurulumu
-setup.intro=openHAB bir\u00e7ok farkl\u0131 eklenti ile gelir. Kolay bir ba\u015flang\u0131\u00e7 i\u00e7in d\u00f6rt adet \u00f6nceden tan\u0131mlanm\u0131\u015f paketleri vard\u0131r. Bu paketler iyi bir se\u00e7enekdir.
-setup.check-doc=Bu paketlerin <a href\="https\://www.openhab.org/docs/configuration/packages.html" target\="_blank">ayr\u0131nt\u0131l\u0131 a\u00e7\u0131klamalar\u0131</a> i\u00e7in \u00e7evrimi\u00e7i belgelere g\u00f6z at\u0131n.
+setup.intro=openHAB bir\u00e7ok farkl\u0131 eklenti ile gelir. Kolay bir ba\u015flang\u0131\u00e7 i\u00e7in d\u00f6rt adet \u00f6nceden tan\u0131mlanm\u0131\u015f paketleri vard\u0131r. Bu paketler iyi bir se\u00e7enektir.
+setup.check-doc=Bu paketlerin <a href\="http\://docs.openhab.org/configuration/packages.html" target\="_blank">ayr\u0131nt\u0131l\u0131 a\u00e7\u0131klamalar\u0131</a> i\u00e7in \u00e7evrimi\u00e7i belgelere g\u00f6z at\u0131n.
```

Comment:
Does anyone know why these links are changed to the old url?

Reply:
Oversight on my side, the strings are fixed over at crowdin now.